content-src/components/TopSites/_TopSites.scss

```diff
@@ -196,6 +197,16 @@ $half-base-gutter: $base-gutter / 2;
     }
   }
 
+  .search-topsite {
+    background-image: url('#{$image-path}glyph-search-16.svg');
+    background-size: 26px;
```

Comment:
I don't think the intention for this is to be different than `$default-icon-size` as defined above, the measurements are just wrong for the spec. Confirming with amy now